reframe/core/pipeline.py

```diff
@@ -1290,7 +1290,7 @@ def run(self):
 
             self.container_platform.validate()
             self.container_platform.mount_points += [
-                (self._stagedir, self.container_platform.RFM_STAGEDIR)
+                (self._stagedir, _STAGEDIR_MOUNT)
```

Comment:
Can't this happen inside the container platform? For example, just before emitting the commands? Or adding it without touching the user mount points at all?

Reply:
We need to know the stage directory of the test to do so.

Reply:
I think the best way is to change the interface of the `emit_prepare_commands()` and the `launch_command()` to accept a path, where you will pass the stagedir.

Reply:
I fixed that.